packages/react-admin/src/Admin.tsx

```diff
@@ -125,32 +127,37 @@ const Admin = (props: AdminProps) => {
     }
 
     return (
-        <AdminContext
-            authProvider={authProvider}
-            basename={basename}
-            dataProvider={dataProvider}
-            i18nProvider={i18nProvider}
-            history={history}
-            customReducers={customReducers}
-            initialState={initialState}
-        >
-            <AdminUI
-                layout={appLayout || layout}
-                dashboard={dashboard}
-                disableTelemetry={disableTelemetry}
-                menu={menu}
-                catchAll={catchAll}
-                theme={theme}
-                title={title}
-                loading={loading}
-                loginPage={loginPage}
-                logout={authProvider ? logoutButton : undefined}
-                ready={ready}
+        <ThemeProvider theme={theme}>
```

Comment:
I think the ThemeProvider should be in the AdminContext. It would facilitate our tests, as we'd only have to wrap mui components with AdminContext (just like we wrap core components with CoreAdminContext).

Reply:
Sure